### PR TITLE
Middle of the Linked List

### LINKED ISSUE #915: Middle element in a linked list
Would like to add an algorithm for finding middle element in linked list in C++.
Please assign this to me under gssoc2021.

### PR DESCRIPTION
**Pull request by tk315**
Kindly check it out and do the needful.
Thanks!

Fixed #915

### CODE REVIEW

File: Code/C++/Middle_of_the_LinkedList.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        j++;
+    }
+
+    cout<<curr->data<<endl;
+
+}
+
+/*
+
+Time Complexity: O(N)
```
Comment (rudrakshi99) [inline Code/C++/Middle_of_the_LinkedList.cpp:83]: add complexities at the end of test cases.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by rudrakshi99** [comment]
Also resolve merge conflicts.

**Comment by rudrakshi99** [comment]
> Also resolve merge conflicts.

Please do again.

**Comment by tk315** [comment]
@rudrakshi99  done

**Pull request merged**